packages/client/src/plugins/elide/adapters/facts.ts

```diff
@@ -65,6 +66,8 @@ type PaginationOptions = {
   after: number;
 };
 
+const PLUGIN_NAMESPACE = 'plugins:elide:facts:adapter';
```

Comment:
Should we have `yavin` somewhere in the namespace?

```suggestion
const PLUGIN_NAMESPACE = 'yavin:plugins:elide:facts:adapter';
```
??

Reply:
https://github.com/yavin-dev/framework/blob/c6cfe6e75ec3f3dcbbbfe9074ca00a1a62232c9f/packages/client/src/models/native-with-create.ts#L70
These extend from the main debug logger (default to yavin:client), so it's still in the name